test/libsolidity/util/TestFileParser.cpp

```diff
@@ -43,6 +43,10 @@ namespace
 	{
 		return isIdentifierStart(c) || isdigit(c);
 	}
+	bool isHex(char c)
```

Comment:
`langutil::isHexDigit`

Reply:
Unfortunately, `isHexDigit` is implemented in an anonymous namespace in the scanner. Plus, it uses hand-written `isDecimalDigit` instead of `std::isdigit`. So, I'd define `isHex` it here, but probably rename it to `isHexDigit`.

Reply:
I would actually argue that our own `isDecimalDigit` might be better than `std::isdigit`: "although some implementations (e.g. Microsoft in 1252 codepage) may classify additional single-byte characters as digits. " - not 100% sure what that means.

If the functions are used multiple times, then they have a right to live in a proper namespace :)

Reply:
Ok, I agree!